internal/services/storage/storage_container_resource_test.go

```diff
@@ -202,6 +433,19 @@ func TestAccStorageContainer_web(t *testing.T) {
 }
 
 func (r StorageContainerResource) Exists(ctx context.Context, client *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	if strings.HasPrefix(state.ID, "/subscriptions") {
+		id, err := commonids.ParseStorageContainerID(state.ID)
+		if err != nil {
+			return nil, err
+		}
+		existing, err := client.Storage.ResourceManager.BlobContainers.Get(ctx, *id)
+		if err != nil {
+			return nil, fmt.Errorf("retrieving %s: %+v", id, err)
+		}
+
+		return pointer.To(existing.Model != nil), nil
+	}
```

Comment:
I think this also wants to have the behaviour split using `!features.FivePointOhBeta` so that it's clear post 5.0 what can and needs to be removed when we clean up 
```suggestion
	if !features.FivePointOhBeta() {
		// the old dance here
	}
	
	id, err := commonids.ParseStorageContainerID(state.ID)
	if err != nil {
		return nil, err
	}
	existing, err := client.Storage.ResourceManager.BlobContainers.Get(ctx, *id)
	if err != nil {
		return nil, fmt.Errorf("retrieving %s: %+v", id, err)
	}

	return pointer.To(existing.Model != nil), nil
```